Rakefile

```diff
@@ -11,7 +11,8 @@ namespace :assets do
   desc 'Update Foundation for Sites assets'
   task update: :clean do
     sh 'bower install'
-    sh 'cp -R bower_components/foundation-sites/js/* vendor/assets/js/'
+    sh 'cp -R bower_components/foundation-sites/dist/js/plugins/* vendor/assets/js/'
```

Comment:
Maybe a non-related question but why does the rails version of this need all the js broken out?  Couldn't we just pull the foundation.js instead? `sh 'cp -R bower_components/foundation-sites/dist/js/foundation.js vendor/assets/js/'`